SQL/Release_patches/20.3_To_21.0_upgrade.sql

```diff
@@ -0,0 +1,325 @@
+    (SELECT BIDSCategoryID FROM bids_category WHERE BIDSCategoryName='dwi'),
+    NULL,
+    (SELECT BIDSScanTypeID FROM bids_scan_type WHERE BIDSSCanType='dwi'),
+    NULL
+  );
+
+SELECT 'Running: SQL/New_patches/2018-08-27-GenderToSex.sql';
+
+ALTER TABLE `candidate` CHANGE COLUMN Gender Sex enum('Male','Female');
+ALTER TABLE `candidate` CHANGE COLUMN ProbandGender ProbandSex enum('Male','Female');
+
+ALTER TABLE `tarchive` CHANGE PatientGender PatientSex varchar(255);
+
+SELECT 'Running: SQL/New_patches/2018-11-05-rename_candidate_CenterID_to_RegistrationCenterID.sql';
+
+ALTER TABLE `candidate` DROP FOREIGN KEY `FK_candidate_1`;
+ALTER TABLE `candidate` 
+    CHANGE COLUMN `CenterID` `RegistrationCenterID` integer unsigned NOT NULL DEFAULT '0';
+ALTER TABLE `candidate`
+  ADD CONSTRAINT `FK_candidate_1` FOREIGN KEY (`RegistrationCenterID`) REFERENCES `psc` (`CenterID`);
+
+
+SELECT 'Running: SQL/New_patches/2018-11-29-Add_instrument_manager_permission.sql';
+
+INSERT INTO permissions (code,description,categoryID) VALUES ('instrument_manager_read','Instrument Manager: View module',(SELECT ID FROM permissions_category WHERE Description = 'Permission'));
+INSERT INTO permissions (code,description,categoryID) VALUES ('instrument_manager_write','Instrument Manager: Install new instruments via file upload',(SELECT ID FROM permissions_category WHERE Description = 'Permission'));
+
+INSERT IGNORE INTO user_perm_rel
+SELECT upr.userID, p.permID FROM user_perm_rel upr JOIN permissions p WHERE upr.permID=1 AND p.code IN ('instrument_manager_read', 'instrument_manager_write');
+
+INSERT INTO LorisMenuPermissions (MenuID, PermID)
+   SELECT m.ID, p.PermID FROM permissions p CROSS JOIN LorisMenu m WHERE p.code='instrument_manager_read' AND m.Label='Instrument Manager';
+INSERT INTO LorisMenuPermissions (MenuID, PermID)
+   SELECT m.ID, p.PermID FROM permissions p CROSS JOIN LorisMenu m WHERE p.code='instrument_manager_write' AND m.Label='Instrument Manager';
+
+SELECT 'Running: SQL/New_patches/2019-01-28_Add_Pwned_Password_ConfigSetting.sql';
+
+-- Adds the option to toggle the usage of the Pwned Passwords
+-- API (https://haveibeenpwned.com/API/v2#PwnedPasswords) in a project. This
+-- is enabled by default to allow for a higher level of security. This
+-- setting is added to allow projects to disable the API check in case of
+-- networking issues.
+INSERT INTO ConfigSettings
+  (
+    Name,
+    Description,
+    Visible,
+    AllowMultiple,
+    DataType,
+    Parent,
+    Label,
+    OrderNumber
+  )
+  SELECT
+    'usePwnedPasswordsAPI',
+    'Whether to query the Have I Been Pwned password API on password changes to prevent the usage of common and breached passwords',
+    1,
+    0,
+    'boolean',
+    ID,
+    'Enable "Pwned Password" check',
+    22
+  FROM
+    ConfigSettings
+  WHERE
+    Name="study";
+
+INSERT INTO Config (ConfigID, Value) SELECT ID, 'true' FROM ConfigSettings WHERE Name="usePwnedPasswordsAPI";
```

Comment:
This should be set to false. I don't think that all instances should send HTTP request to a server in Netherlands at each password changes.

Reply:
hmm... I guess it was approved by someone.

I track that and see if it was previously discussed

Reply:
It was discussed.. @xlecours is right, it's probably just that the schema was reviewed/changed and the patch was missed.

Reply:
I was mistaken, the discussion was about whether to include a config setting, not what the default should be.